src/SDKs/Azure.Core/data-plane/Azure.Core/Pipeline/SynchronousHttpPipelinePolicy.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Azure.Core.Pipeline
+{
+    public abstract class SynchronousHttpPipelinePolicy: HttpPipelinePolicy
```

Comment:
Apologies if I'm missing something obvious, are we trying to denote that this pipeline runs each step in order?  Seeing as this allows for both synchronous and asynchronous processing, should we consider something like `SequentialHttpPipelinePolicy`?

Reply:
It's more like "I don't care about IO" policy. It was named `IOAgnostic` in my first iteration but that was a pretty terrible name. I think `Sequential` is slightly confusing too.